src/libawkward/array/RecordArray.cpp

```diff
@@ -352,7 +352,7 @@ namespace awkward {
   }
 
   const std::shared_ptr<Content> RecordArray::flatten(int64_t axis) const {
-    throw std::runtime_error("FIXME: not implemented");
+    throw std::invalid_argument("RecordArray is one-dimentional");
```

Comment:
Perhaps just `"RecordArray cannot be flattened"` because a RecordArray can contain arrays in one or all of its fields, and some user is likely to be confused by the statement that it's one-dimensional. "Number of dimensions" becomes a nebulous concept for arbitrary data structures.